src/content/advanced/gitops/apps/add_app_template.md

```diff
@@ -0,0 +1,99 @@
+---
+linkTitle: Add a new App Template
+title: Add a new App Template
+description: Learn how to create application templates to deploy apps using GitOps.
+weight: 50
+menu:
+  main:
+    identifier: advanced-gitops-apps-add-app-template
+    parent: advanced-gitops-apps
+last_review_date: 2023-02-10
+user_questions:
+  - How can I create an template for app deployment in GitOps?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-honeybadger
+---
+
+# Add a new App Template to the repository
+
+In order to avoid adding the same application from scratch across all your clusters, you can prepare App Templates that provide a pre-configured version of an App. This also allows you to manage and version an app's config even if the app itself is not yet installed in any cluster.
```

Comment:
```suggestion
In order to avoid adding the same application from scratch across all your clusters, you can prepare App Templates providing a pre-configured version of an App. This also allows you to manage and version an app's configuration even if the app itself is not yet installed in any cluster.
```